src/examples/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ set( contact_examples
      common_plane_gpu.cpp
      common_plane.cpp
      mortar_lm_patch_test.cpp
+     neohookean_timing.cpp
```

Comment:
We will need to turn this test off when Tribol is built without Enzyme.  Try removing this from the list and adding this  after: `blt_list_append(TO contact_examples ELEMENTS neohookean_timing.cpp IF TRIBOL_USE_ENZYME )`